spec/logic_spec.rb

```diff
@@ -0,0 +1,48 @@
+require './lib/logic'
```

Comment:
# OPTIONAL CHNAGES: ♻️ 
```suggestion
require_relative '../lib/logic'
```

- [ ] Using require_relative for files in your directory instead of `require` is a ruby convention. Also, add an extra period to import the file to your test suites correctly.